numpy/core/src/umath/simd.inc.src

```diff
@@ -1344,8 +1379,12 @@ static NPY_GCC_OPT_3 NPY_GCC_TARGET_@ISA@ void
 
         if (num_remaining_elements < num_lanes)
             load_mask = @isa@_get_partial_load_mask(num_remaining_elements,
-                                                         num_lanes);
-        @vtype@ x  = @isa@_masked_load(load_mask, ip);
+                                                    num_lanes);
+        @vtype@ x;
+        if (stride == 1)
```

Comment:
always put `{ }` around loops and conditions also if it is only one line

Reply:
fixed this.

Reply:
sorry should have been more specific the numpy style is:
```
if () {
}
else {
}
```

Reply:
ah sorry about that, should be more consistent now.